versions/scylla/1.15.3/patch

```diff
@@ -0,0 +1,5827 @@
+diff --git a/.github/actions/setup-environment/action.yml b/.github/actions/setup-environment/action.yml
+index f68ffec..5aa4c06 100644
+--- a/.github/actions/setup-environment/action.yml
```

Comment:
Please keep changes only related to the tests, github workflows are not relevant